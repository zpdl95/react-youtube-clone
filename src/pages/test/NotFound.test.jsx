import React from 'react';
import { Route } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { withRouter } from '../../test/utils';
import { NotFound } from '../index';

describe('NotFound', () => {
  it('NotFound snapshot', () => {
    const component = renderer.create(
      withRouter(<Route path='/' element={<NotFound />} />)
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
