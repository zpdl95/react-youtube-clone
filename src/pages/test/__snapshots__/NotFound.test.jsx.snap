// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`NotFound > NotFound snapshot 1`] = `
<div>
  NotFound
</div>
`;
